src/BIAnalyticsManagementService/CreateParticipantScreeningProfile/CreateParticipantScreeningProfile.cs

```diff
@@ -18,44 +17,25 @@ public CreateParticipantScreeningProfile(ILogger<CreateParticipantScreeningProfi
         _httpRequestService = httpRequestService;
     }
 
-    [Function("CreateParticipantScreeningProfile")]
-    public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "post")] HttpRequestData req)
+    [Function(nameof(CreateParticipantScreeningProfile))]
+    public async Task Run([EventGridTrigger] EventGridEvent eventGridEvent)
     {
         _logger.LogInformation("Create Participant Screening Profile function start");
 
-        string nhsNumber = req.Query["nhs_number"];
-
-        if (string.IsNullOrEmpty(nhsNumber))
-        {
-            _logger.LogError("nhsNumber is null or empty.");
-            return req.CreateResponse(HttpStatusCode.BadRequest);
-        }
-
-        var baseParticipantUrl = Environment.GetEnvironmentVariable("GetParticipantUrl");
```

Comment:
Now that we're no longer using this environment variable, it can be removed from local settings and compose files